presto-orc/src/main/java/com/facebook/presto/orc/OrcOutputBuffer.java

```diff
@@ -461,13 +456,21 @@ private void writeChunkToOutputStream(byte[] chunk, int offset, int length)
 
         int compressedSize = compressor.compress(chunk, offset, length, compressionBuffer, 0, compressionBuffer.length);
         if (compressedSize < length) {
+            if (dwrfEncryptor.isPresent()) {
```

Comment:
It's unclear to me how you handle encrypted but uncompressed file. Would be good to throw in that case if not supported at the moment

Reply:
As per offline conversation I will add a follow up to support this.  I previously thought that wasn't supported by the file format.